Symfony/CS/Fixer/Symfony/PhpdocParamsFixer.php

```diff
@@ -26,6 +26,9 @@ class PhpdocParamsFixer extends AbstractFixer
     private $regex;
     private $regexCommentLine;
 
+    /**
+     * Create a new phpdoc params fixer instance.
+     */
```

Comment:
> our rule in Symfony is to avoid meaningless phpdoc (a recent PR is even removal the constructor phpdoc when it contains only typehinted params and we don't have anything else to document)

So in following code:
```php
/**
 * Set token prorotype.
 *
 * @param array|string|Token $prototype
 */
function setPrototype($prototype) {...}
```

The docs should be removed?
How can I know then what types for param are allowed?